Tests/FluentAssertions.Equivalency.Specs/DictionarySpecs.cs

```diff
@@ -494,6 +494,20 @@ public void When_a_generic_dictionary_is_typed_as_object_it_should_respect_the_r
         act.Should().NotThrow();
     }
 
+    [Fact]
+    public void When_a_generic_dictionary_is_typed_as_object_without_nesting_it_should_respect_the_runtime_typed()
```

Comment:
🤔 This test also succeeds when you omit `ExcludingNestedObjects`, so I'm not sure what it is supposed to do.